packages/carbon-graphs/dev/data/dataObjects/Line/helper.js

```diff
@@ -0,0 +1,26 @@
+const createGraphValuesWithSin = (
+  length, initialX, xModifier, yModifier,
```

Comment:
i think here's a case where having the parameters as an object will be helpful to state what each value is for, when calling the function:
```suggestion
const createGraphValuesWithSin = ({ length, initialX, xModifier, yModifier }) => {
```

```js 
// example of calling function
createGraphValuesWithSin({ length: 2, initialX: 3, xModifier: 2, yModifier: Math.PI });
```

Reply:
Changed to [object](https://github.com/cerner/terra-graphs/pull/106/commits/2fcc3417f78ac531d21f8c9f065013822d51447c#diff-9291820bed2ddd5388c7b9232d223bb579ec51ad71caae5af9ec996a2a0f2bfaR18).